packages/valaxy-theme-yun/components/YunPostMeta.vue

```diff
@@ -31,14 +30,14 @@ const siteConfig = useSiteConfig()
     flex="~ col" justify="center" items="center" text="sm" py="1"
   >
     <div v-if="frontmatter.date" class="post-time flex items-center">
-      <span class="posted-time inline-flex-center" :title="t('post.posted') + formatISO(frontmatter.date)">
+      <span class="posted-time inline-flex-center" :title="t('post.posted') + formatDate(frontmatter.date, 'yyyy-MM-dd HH:mm:ss')">
```

Comment:
Merge in one util?

Reply:
Do you mean abstracting `formatDate(date, 'yyyy-MM-dd HH:mm:ss') into a function?